splash/browser_tab.py

```diff
@@ -672,7 +671,7 @@ def html(self):
         """ Return HTML of the current main frame """
         self.logger.log("getting HTML", min_level=2)
         frame = self.web_page.mainFrame()
-        result = bytes(frame.toHtml().encode('utf-8'))
+        result = frame.toHtml().encode('utf-8')
```

Comment:
It is a bit strange that some methods return bytes and some methods return unicode. E.g. `set_contents` requires unicode, but `html`returns bytes. 

toHtml returns QString, which is basically qt's unicode, right? Why are you encoding it to utf8 here?